src/Flysystem/FlysystemAssetStore.php

```diff
@@ -417,21 +418,23 @@ protected function moveBetweenFilesystems($fileID, Filesystem $from, Filesystem
 
     public function grant($filename, $hash)
```

Comment:
I can't quite parse your sentence, but I assume you mean changing the request signature to `grant($filename, $hash, $session)`? I think that's a good idea, do you want to create a beta1 issue?